### PR TITLE
Add testcases model and admin

### PR DESCRIPTION
**Pull request by yothinix**
Change proposed in this pull request:
- Add `testcases` app
- Add model `TestCase`
- Register admin for `TestCase` model
- Install `model_mommy` in dependencies
- Change coverage to ignore `*wsgi.py` and ``*manage.py`
- Add `python_classes` option in pytest.ini to look for *Test class for Test class only

### CODE REVIEW

File: api/prontolista/testcases/tests/test_models.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+
+from model_mommy import mommy
+
+from projects.models import Project
+from ..models import TestCase as TestCaseModel
```
Comment (zkan) [inline api/prontolista/testcases/tests/test_models.py]: Let's switch this with `from projects.models import Project`?

File: api/pyproject.toml [hunk 2]
```diff
@@ -12,6 +12,7 @@ psycopg2 = "^2.8"
 sentry-sdk = "^0.12.3"
 django-extensions = "^2.2"
 django-waffle = "=0.17.0"
+model_mommy = "=1.6.0"
```
Comment (zkan) [inline api/pyproject.toml]: This should be in the dev dependencies. :D

File: api/prontolista/testcases/tests/test_models.py (hunk 1 above)
Comment (yothinix) [reply api/prontolista/testcases/tests/test_models.py]: Will fix this after we complete setup isort kub

File: api/pyproject.toml (hunk 2 above)
Comment (yothinix) [reply api/pyproject.toml]: Moved Laew

**Pull request merged**